### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by Bekovmi**
1. Fix bugs with download dataset.
2. Fix bug with experiment

### CODE REVIEW

File: src/model.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+
+import torch
+import torch.nn as nn
+from catalyst.dl import registry
+
+
+@registry.Model
```
Comment (ngxbac) [inline src/model.py:7]: I think we do not need register model here since we already do it in `__init__.py`

File: config.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      loss_CE:                               
+        callback: CriterionCallback             
+        input_key: labels                         
+        output_key: labels
+        prefix: &loss CrossEntropyLoss                  
+      
+      accuracy:                                 
+        callback: DiceCallback                  
+        input_key: labels
+        output_key: labels
```
Comment (ngxbac) [inline config.yml:41]: It is not correct.  are they `labels` and `logits` ?

File: config.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      weight_decay: 0.0003                      
+    scheduler_params:                           
+      scheduler: MultiStepLR                    
+      milestones: [10]                          
+      gamma: 0.3                                
+    callbacks_params:                           
+      loss_CE:                               
+        callback: CriterionCallback             
+        input_key: labels                         
+        output_key: labels
```
Comment (ngxbac) [inline config.yml:35]: It is not correct.  are they `labels` and `logits` ?
